### PR TITLE
match navigation and footer in public pages

### LINKED ISSUE #5572: Match header nav styles
Apart that the font is different in /explore section, we should match both styles. cartodb should looks like website.

cartodb:
image

website:
image

### PR DESCRIPTION
**Pull request by matallo**
close #5572

I made shortcuts to get to the end result of this, but at this time, unless there's a big change in public pages layout I won't invest much more time in this, new styles are mostly copypaste from commercial site (I'll ignore hound warnings)

- [x] public map
- [x] data-library
- [x] explore

### CODE REVIEW

**Comment by matallo** [comment]
public map

fb5e0499-d4b6-464f-9006-7acd96ac13b4
e0554c30-c493-4524-8cce-4b2cc28a8868

**Comment by matallo** [comment]
CR @javierarce

File: app/assets/stylesheets/common/footer.css.scss [hunk 1]
```diff
@@ -43,117 +35,110 @@
...
 .Footer-listItem {
   display: block;
   margin-top: $sMargin-element;
   text-align: center;
 }
+.Footer-columns {
+  border-top: 1px solid #EEE;
+  padding-bottom: 60px;
+}
+.Footer-columns--noborder {
```
Comment (javierarce) [inline app/assets/stylesheets/common/footer.css.scss]: Little detail: shouldn't this be `Footer-columns--noBorder` instead?

File: app/assets/stylesheets/common/grid.css.scss [hunk 2]
```diff
@@ -51,6 +51,16 @@
   @include justify-content(space-between);
 }
 
+.Grid-inner--shortpadding {
```
Comment (javierarce) [inline app/assets/stylesheets/common/grid.css.scss]: ditto: maybe would be better written as `Grid-inner--shortPadding`?

**Comment by javierarce** [comment]
Except for the hound warnings and a couple of details I mentioned, code looks :cool:

**Comment by matallo** [comment]
"Begin pseudo elements with double colons: ::" should be fixed at the iconfont generator, fixed the other warning, merging then

**Pull request merged**